### ISSUE TITLE
RandAugment magnitude:  10 in paper while 5 in code

### ISSUE THREAD

**Issue by ShoufaChen** [deepmind/deepmind-research#274]
Hi, 

Thanks for your great work.
I noticed that the `magnitude` of RandAugment is 10 in the paper Appendix.H while 5 in the code:

https://github.com/deepmind/deepmind-research/blob/769bfdbeafbcb472cb8e2c6cfa746b53ac82efc2/perceiver/train/experiment.py#L192

I was wondering which value is correct?

Thanks in advance.

**Comment by djaegle** [comment]
Hi, thanks for noticing and flagging this! We used RandAugment with a magnitude of 5 in all ImageNet experiments here. The value of 10 given in the paper is a typo: we'll fix it in the next draft we release.

**Comment by ShoufaChen** [comment]
Hi, @djaegle 

Thanks for your clarification.

A similar question about learning rate here:

https://github.com/deepmind/deepmind-research/blob/769bfdbeafbcb472cb8e2c6cfa746b53ac82efc2/perceiver/train/experiment.py#L88

The paper said the 'a flat learning rate of 2e-3'.

I was also wondering which value is correct?
